securedrop_client/gui/widgets.py

```diff
@@ -2510,6 +2515,21 @@ def center_dialog(self):
         y_center = (application_window_size.height() - dialog_size.height()) / 2
         self.move(x + x_center, y + y_center)
 
+    def animate_activestate(self):
+        self.continue_button.setIcon(QIcon(self.animation.currentPixmap()))
```

Comment:
Just a note: If we use `setIcon` to add an icon to the button, it will appear smaller than the design, but I think this is a good first step. Eventually we'll need to make our own QPushButton class that allows for larger icons, also see: https://github.com/freedomofpress/securedrop-client/pull/679#issuecomment-570636850

Reply:
Preach it sister..! :slightly_smiling_face: 

Small steps, etc...